src/main/java/io/vertx/ext/hawkular/MetricsTypeEnum.java

```diff
@@ -0,0 +1,46 @@
+package io.vertx.ext.hawkular;
+
+import io.vertx.codegen.annotations.VertxGen;
+
+/*
```

Comment:
Please update your IDE format and template settings. See https://github.com/vert-x3/wiki/wiki/Vert.x-code-style-guidelines

The indentation is two spaces, not four. The copyright notice must be at the top of the file.